compiler/rustc_expand/src/mbe/diagnostics.rs

```diff
@@ -212,6 +218,14 @@ impl<'matcher> Tracker<'matcher> for FailureForwarder {
     fn description() -> &'static str {
         "failure-forwarder"
     }
+
+    fn set_expect_tok(&mut self, tok: &'matcher Token) {
+        self.0 = Some(tok);
+    }
+
+    fn get_expect_tok(&self) -> Option<&'matcher Token> {
```

Comment:
A rename would probably be better
```suggestion
    fn set_expected_token(&mut self, tok: &'matcher Token) {
        self.0 = Some(tok);
    }

    fn expected_token(&self) -> Option<&'matcher Token> {
```